NuRadioMC/test/examples/test_examples.sh

```diff
@@ -4,16 +4,26 @@ mkdir -p output
 python3 T01generate_event_list.py
 python3 T02RunSimulation.py 1e19_n1e3.hdf5 surface_station_1GHz.json config.yaml output/output.hdf5 output.nur
 python3 T03visualizeVeff.py
+rm 1e18_n1e4.hdf5
+rm 1e19_n1e3.hdf5
+rm Veff.pdf
+rm limits.pdf
+rm output.nur
+rm -r output
 
-cd ../02_DnR
-python3 E01detector_simulation.py event_input/1e19_n1e3comparison1.hdf5 detector/string_to_100m.json config.yaml output.hdf5
-python3 A01analyse_simulation.py output.hdf5
+#cd ../02_DnR
```

Comment:
Why do we stop testing DnR?

Reply:
The example does not work anymore because the hdf5 structure changed. Before it was implemented in a complicated way that involved the execution of a custom module. With the new structure this analysis is easier but it would still be some work to re implement this analysis. As we don't really need this at the moment, I rather wanted to remove the example. I should also remove it from the repo completely and not just the testing